whois-commons/src/main/resources/nrtm_client_schema.sql

```diff
@@ -59,7 +59,7 @@ CREATE TABLE `last_mirror`
     `object_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
     `sequence_id` int(10) unsigned NOT NULL DEFAULT '1',
     `timestamp` int(10) unsigned NOT NULL DEFAULT '0',
-    `object_type` tinyint(3) unsigned NOT NULL DEFAULT '0',
+    `object_type` varchar(254) NOT NULL DEFAULT '0',
```

Comment:
Why are you not using `tinyint(3)` like the last and history tables?

Reply:
I think that storing the ID is something internal for us.
I am storing the object name instead of the object type.

I can use ObjectTypeIds and get the id of the object. But this means, that I am using something of whois that the client may be doesn't have (I would be using whois-common module)

Reply:
You should be using the whois schema to store RPSL objects, not duplicating it. How does the NRTMv3 client write RPSL into the mirror database?

Reply:
maybe we can discuss this with Mahesh?

Reply:
this will be done as part of [DB-5870](https://jira.ripe.net/browse/DB-5870)